lib/src/main/java/com/otaliastudios/transcoder/internal/thumbnails/DefaultThumbnailsEngine.kt

```diff
@@ -316,17 +316,18 @@ class DefaultThumbnailsEngine(
 
     override fun removePosition(sourcePath: String, sourceId: String, positionUs: Long) {
         if (positionUs < 0) {
-            val activeStub = stubs.firstOrNull()?.takeIf { it.request.sourceId() == sourceId }
+//            val activeStub = stubs.firstOrNull()?.takeIf { it.request.sourceId() == sourceId }
```

Comment:
what happens if that stub is being processed?

Reply:
system will stop processing it and move on the next item

Reply:
how?